react/src/lib/SpaceListItem/index.js

```diff
@@ -170,15 +181,14 @@ class SpaceListItem extends React.PureComponent {
                         </ListItemSection>
                       )}
                     </div>
-                    {attachments &&
-                      attachments.length && (
-                        <div className="md-list-item__attachment--bottom">
+                {attachments && attachments.length && (
+                  <div className={'md-list-item__attachment--bottom' + `${(theme && ` ${theme}`) || ''}`}>
                           {attachments[0]}
                         </div>
                     )}
                   </ListItemSection>
                 ) : (
-                  <div className="md-list-item__result">
+              <div className={'md-list-item__result' + `${(theme && ` ${theme}`) || ''}`}>
```

Comment:
since the theme classname should be defined top level of the component, we shouldn't need to add classnames to all the children. the css selectors should be able to handle that